src/test/kotlin/com/terraformation/backend/seedbank/model/accession/AccessionModelCalculationsTest.kt

```diff
@@ -1982,149 +1140,4 @@ internal class AccessionModelTest {
       assertEquals(grams(5), accession.estimatedWeight)
     }
   }
-
-  @Nested
-  inner class V1ToV2Conversion {
```

Comment:
I'm not seeing where this moved. I see the v2-to-v1 tests but not these.

Reply:
good catch! I thought that I'd moved them, but looks like I missed these 😞